performance-tests/crypto-sidecar/index.js

```diff
@@ -0,0 +1,53 @@
+const Koa = require("koa");
+const crypto = require("@nmshd/crypto");
+const koaBody = require("koa-body").koaBody;
+
+const app = new Koa();
+
+const port = 3000;
+
+app.use(
+    koaBody({
+        jsonLimit: "1kb"
+    })
+);
+
+app.use(async (ctx, next) => {
+    const start = Date.now();
+    await next();
+    const ms = Date.now() - start;
+    console.log(`${ctx.method} ${ctx.url} - ${ms} ms`);
+});
+
+app.use(async function (ctx, next) {
+    if (ctx.method !== "GET" || ctx.path !== "/keypair") return await next();
+
+    ctx.body = JSON.stringify(await crypto.CryptoSignatures.generateKeypair());
+    ctx.status = 200;
```

Comment:
I'm no expert in JS/TS, but I would suggest adding a `ctx.type = "application/json";`, so that the response has an explicit type

Reply:
Great idea, ty!